<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>466</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>928</width>
    <height>466</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>938</width>
    <height>476</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Selecionar intervalo de datas para estatísticas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(216, 164, 194);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtnConfirmar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {     
	background-color: rgb(0, 170, 0);
	border-radius: 10px;
}
QPushButton:hover {   
	background-color: rgb(0, 190, 0);
}
QPushButton:pressed {   
	background-color: rgb(0, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>751</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Informe o intervalo de datas para consultar os dados estatísticos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="DataInicial">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>21</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-color: rgb(0,0,0);
font: 21pt &quot;Ubuntu&quot;;
color: black;
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2021</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="timeSpec">
     <enum>Qt::LocalTime</enum>
    </property>
   </widget>
   <widget class="QDateEdit" name="DataFinal">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>21</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-color: rgb(0,0,0);
font: 21pt &quot;Ubuntu&quot;;
color: black;
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2021</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="timeSpec">
     <enum>Qt::LocalTime</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data inicial</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data final</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../IMAGENS/PicsArt_10-18-10.18.32.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>BtnConfirmar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
